pkg/pipelines/pipelines.go

```diff
@@ -21,6 +21,7 @@ import (
 	clipr "github.com/tektoncd/cli/pkg/cmd/pipelinerun"
 	"github.com/tektoncd/cli/pkg/options"
 	prsort "github.com/tektoncd/cli/pkg/pipelinerun/sort"
+	v1 "github.com/tektoncd/pipeline/pkg/apis/pipeline/v1"
```

Comment:
I'm not sure what are the best practices for this and I agree that it's a mess in our codebase but having both `v1` and `metav1` in our file doesn't look right. We should use one style. Do we actually need both `v1beta1` and `v1` here?

Reply:
v1beta1 is deprecated and will remove it.